.github/workflows/docker-ci.yaml

```diff
@@ -24,6 +24,9 @@ env:
 jobs:
   build:
     runs-on: ubuntu-16.04
+    strategy:
+      matrix:
+        es: [6, 7]
```

Comment:
This way makes the CI name no clear. Please try to make to `docker-ci / build(es6)` at least.